src/main/webapp/signup.html

```diff
@@ -18,7 +18,7 @@
     
     <title>Sign Up | Mintern</title>
   </head>
-  <body onload="isUserRegistered()">
+  <body onload="loadSignup()">
```

Comment:
Use the onBodyLoad() function to set things since the loading of page, since that is the one being used across other PRs. You can add the loadSignup function inside the onBodyLoad() to avoid merge conflicts.

Reply:
This one is different, though. This is like the onBodyLoad() but for the signup page, which loads completely different functions.

Reply:
Which leads me to the suggestion of changing the name of onBodyLoad() to loadIndex(), for function names to be consistent, what do you think?

Reply:
Sounds great, please do so!

Reply:
Done!